src/js/elements/location_info_box.js

```diff
@@ -1,5 +1,6 @@
-import {Observable, numFmt} from '../utils';
+import {Observable, numFmt, formatNumericalValue} from '../utils';
```

Comment:
I like the DRYing up with refactoring this out into a util method. Well done.

Reply:
thanks